torch/csrc/autograd/VariableTypeUtils.h

```diff
@@ -88,6 +88,11 @@ inline void increment_version(Tensor & t) {
   impl::bump_version(t);
 }
 
+inline void increment_version(at::TensorList tensors) {
```

Comment:
so won't the version counter be different depending on if we take the fast path or not?

Reply:
its gone